src/go/configinfo/service_info.go

```diff
@@ -952,15 +955,6 @@ func (s *ServiceInfo) processTypes() error {
 	return nil
 }
 
-// Get the MethodInfo by full name. Prefer to use this function when getting methods,
-// as it outputs an actionable error message.
-func (s *ServiceInfo) getMethod(name string) (*MethodInfo, error) {
-	if s.Methods[name] == nil {
```

Comment:
Are we sure we don't want this?

Are there cases: 
- we want to raise the error if it shouldn't happen
- we want to log the api method if it is OK some api methods aren't defined

Reply:
> we want to raise the error if it shouldn't happen

It should never happen in ESPv2 because the API compiler already does this validation, as mentioned above.

> we want to log the api method if it is OK some api methods aren't defined

I agree this is a good case. But unfortunately many callers are using `ServiceInfo::Methods` directly, as seen in the screenshot I messaged you.

Since I will remove ServiceInfo soon, let's remove this behavior for now, and we can add logging in the replacement when generating routes.